src/jdk.jlink/share/classes/jdk/tools/jlink/internal/plugins/SystemModulesPlugin.java

```diff
@@ -587,440 +574,407 @@ private void dedups(ModuleDescriptor md) {
         /**
          * Generate SystemModules class
          */
-        public ClassWriter getClassWriter(Configuration cf) {
-            ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_MAXS
-                                             + ClassWriter.COMPUTE_FRAMES);
-            cw.visit(Opcodes.V1_8,
-                     ACC_FINAL+ACC_SUPER,
-                     className,
-                     null,
-                     "java/lang/Object",
-                     new String[] { "jdk/internal/module/SystemModules" });
-
-            // generate <init>
-            genConstructor(cw);
+        public byte[] getClassWriter(Configuration cf) {
```

Comment:
This method no longer returns a class writer and should be renamed.   What about `genClassBytes`?

Reply:
renamed to `genClassBytes` as proposed, thanks.